### PR TITLE
Update Valheim Plus

### PR DESCRIPTION
**Pull request by gOOvER**
# Description

- update to PTDL_v2
- update to new Valheim image
- added crossplay
- updated Steam install script to latest

## Checklist for all submissions

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/parkervcp/eggs/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Have you tested and reviewed your changes with confidence that everything works?
* [x] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?:

### CODE REVIEW

**Comment by QuintenQVD0** [comment]
I personaly prefer this PR over #1907

File: game_eggs/steamcmd_servers/valheim/valheim_plus/egg-valheim-plus-mod.json [hunk 1]
```diff
@@ -1,21 +1,19 @@
...
     "name": "Valheim Plus Mod",
     "author": "info@goover.de",
     "description": "A brutal exploration and survival game for 1-10 players, set in a procedurally-generated purgatory inspired by viking culture.",
-    "features": [
-        "steam_disk_space"
-    ],
-    "images": [
-        "ghcr.io\/parkervcp\/games:source"
-    ],
+    "features": null,
```
Comment (QuintenQVD0) [inline game_eggs/steamcmd_servers/valheim/valheim_plus/egg-valheim-plus-mod.json:11]: Please add back the: steam_disk_space

**Review by QuintenQVD0 — CHANGES_REQUESTED** [review]

File: game_eggs/steamcmd_servers/valheim/valheim_plus/egg-valheim-plus-mod.json (hunk 1 above)
Comment (gOOvER) [reply game_eggs/steamcmd_servers/valheim/valheim_plus/egg-valheim-plus-mod.json]: done

**Comment by QuintenQVD0** [comment]
@gOOvER are those errors normal on startup: https://pastebin.com/QdSUFUXf ?

**Comment by gOOvER** [comment]
> @gOOvER are those errors normal on startup: [pastebin.com/QdSUFUXf](https://pastebin.com/QdSUFUXf) ?

Yes, thats normal errors on Mono

**Review by QuintenQVD0 — APPROVED** [review]

**Pull request merged**